lib/plugins/filter/before_post_render/backtick_code_block.js

```diff
@@ -12,6 +12,7 @@ function backtickCodeBlock(data) {
   if (!config.enable) return;
   data.content = data.content.replace(rBacktick, ($0, start, $2, _args, content, end) => {
     const args = _args.split('=').shift();
+    const exclude_languages = config.exclude_languages || [];
```

Comment:
need to move this into L17-22 for consistency.
add a null checking `if (options.exclude_languages)` before `if (exclude_languages.includes(args)) {}`.

Reply:
`exclude_languages` also need to be added to `lib/hexo/default_config.js`.
see https://github.com/hexojs/hexo/pull/3827 for example.

Reply:
Fixed.